Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -4295,20 +4295,17 @@ void WebPageProxy::sendWheelEvent(WebCore::FrameIdentifier frameID, const WebWhe
     internals().wheelEventActivityHysteresis.impulse();
 #endif
 
-    if (!hasRunningProcess())
-        return;
-
     Ref process = processContainingFrame(frameID);
     if (protectedDrawingArea()->shouldSendWheelEventsToEventDispatcher()) {
         sendWheelEventScrollingAccelerationCurveIfNecessary(frameID, event);
-        process->protectedConnection()->send(Messages::EventDispatcher::WheelEvent(webPageIDInProcess(process), event, rubberBandableEdges), 0, { }, Thread::QOS::UserInteractive);
```

Comment:
Maybe the reason to use connection directly was to be able to pass the QOS?

Reply:
No, AuxiliaryProcessProxy uses `QoS::UserInteractive` for all messages. So that's the default QoS.